pkg/agent/openflow/client.go

```diff
@@ -630,8 +636,8 @@ func (c *client) InstallServiceGroup(groupID binding.GroupIDType, withSessionAff
 func (c *client) UninstallGroup(groupID binding.GroupIDType) error {
 	c.replayMutex.RLock()
 	defer c.replayMutex.RUnlock()
-	if !c.bridge.DeleteGroup(groupID) {
-		return fmt.Errorf("group %d delete failed", groupID)
+	if err := c.bridge.DeleteGroup(groupID); err != nil {
+		return fmt.Errorf("error when deleting Service Endpoints Group: %v", err)
```

Comment:
```suggestion
		return fmt.Errorf("error when deleting Service Endpoints Group %d: %v", groupID, err)
```
It doesn't make sense to remove the groupID here.

Reply:
You mean that we should pass the group struct here?

Reply:
Got it, thanks.